framework/src/test/java/org/tron/core/exception/TronErrorTest.java

```diff
@@ -57,7 +59,16 @@ public void testTronError() {
   }
 
   @Test
-  public void ZksnarkInitTest() {
+  public void ZksnarkInitTest() throws IllegalAccessException, NoSuchFieldException {
+    Field initializedField = ZksnarkInitService.class.getDeclaredField("initialized");
+    initializedField.setAccessible(true);
+    Field modifiersField = Field.class.getDeclaredField("modifiers");
+    modifiersField.setAccessible(true);
+    modifiersField.setInt(initializedField,
+        initializedField.getModifiers() & ~java.lang.reflect.Modifier.FINAL);
+    AtomicBoolean atomicBoolean = (AtomicBoolean) initializedField.get(null);
+    atomicBoolean.set(false);
+
     try (MockedStatic<JLibrustzcash> mock = mockStatic(JLibrustzcash.class)) {
```

Comment:
Is a try-finally statement needed here to restore the original value for `initialized`?